clients/go/gengo.go

```diff
@@ -62,10 +62,10 @@ import (
 
 		"go.opentelemetry.io/otel"
 		"go.opentelemetry.io/otel/propagation"
-		"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
+		"go.opentelemetry.io/otel/sdk/trace/tracetest"
 		"go.opentelemetry.io/otel/sdk/resource"
 		sdktrace "go.opentelemetry.io/otel/sdk/trace"
-		semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
+		semconv "go.opentelemetry.io/otel/semconv/v1.10.0"
```

Comment:
Whoa, super weird, I wonder why they decided to name their modules this way. I guess allowing different version of their specification to work in the same Go module or at least not tying the Go mod version to the spec version.

Reply:
Yeah, not a fan of that at all.